### PR TITLE
Develop nazar feature/notes

### PR DESCRIPTION
**Pull request by Nevrik-core**

### CODE REVIEW

File: Notes/notes.py [hunk 1]
```diff
@@ -1,18 +1,52 @@
...
+    def edit_note(self, title):
         pass
+
     
+
     def delete(self, name):
-        pass
+        if name in self.data:
+            del self.data[name]
+            print(f"Note with title '{name}' has been deleted.")
```
Comment (AegisVP) [inline Notes/notes.py:32]: delete print()

File: Notes/notes.py [hunk 2]
```diff
@@ -1,18 +1,52 @@
...
+        else:
+            print(f"Note with title '{name}' not found.")
```
Comment (AegisVP) [inline Notes/notes.py:34]: delete print()

File: Notes/notes.py (hunk 1 above)
Comment (AegisVP) [reply Notes/notes.py:32]: return true

**Review by AegisVP — CHANGES_REQUESTED** [review]

File: Notes/notes.py (hunk 2 above)
Comment (AegisVP) [reply Notes/notes.py:34]: raise KeyError(f"Note with title '{name}' not found.")

**Review by DeWince — APPROVED** [review]

**Pull request merged**